<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="XSun" />

<meta name="date" content="2024-06-28" />

<title>Differential expression &amp; genotype-expression correlation of PICALM in AD, nonAD and all</title>

<script src="site_libs/header-attrs-2.25/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.22/datatables.js"></script>
<link href="site_libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">PICALM</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xsun1229/PICALM">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Differential expression &amp;
genotype-expression correlation of PICALM in AD, nonAD and all</h1>
<h4 class="author">XSun</h4>
<h4 class="date">2024-06-28</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-06-28
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>PICALM/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20240628code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20240628)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20240628code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20240628)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomxsun1229PICALMtree4fbceb130ae35dcebb7c13bb488a304b72b5201etargetblank4fbceb1a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/xsun1229/PICALM/tree/4fbceb130ae35dcebb7c13bb488a304b72b5201e" target="_blank">4fbceb1</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomxsun1229PICALMtree4fbceb130ae35dcebb7c13bb488a304b72b5201etargetblank4fbceb1a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/xsun1229/PICALM/tree/4fbceb130ae35dcebb7c13bb488a304b72b5201e" target="_blank">4fbceb1</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Unstaged changes:
    Modified:   analysis/geno_expr_cor.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/geno_expr_cor.Rmd</code>) and
HTML (<code>docs/geno_expr_cor.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xsun1229/PICALM/blob/fa92d8cc7a7543b68b9504caa8f9f8cb12b9bee2/analysis/geno_expr_cor.Rmd" target="_blank">fa92d8c</a>
</td>
<td>
XSun
</td>
<td>
2024-06-28
</td>
<td>
update
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xsun1229/PICALM/blob/afc02215f317fc21701a1f1e42bd4318631500ac/analysis/geno_expr_cor.Rmd" target="_blank">afc0221</a>
</td>
<td>
XSun
</td>
<td>
2024-06-28
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xsun1229/PICALM/afc02215f317fc21701a1f1e42bd4318631500ac/docs/geno_expr_cor.html" target="_blank">afc0221</a>
</td>
<td>
XSun
</td>
<td>
2024-06-28
</td>
<td>
update
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>library(ggplot2)
library(dplyr)</code></pre>
<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>We address the one of the comment from the reviewer here.</p>
<p>1.postmortem brain microglia AD vs. nonAD differential expression
analysis of PICALM</p>
<ol start="2" style="list-style-type: decimal">
<li>genotype-expression correlation of PICALM.</li>
</ol>
<pre class="r"><code>df &lt;- readxl::read_excel(&quot;/project2/xinhe/xsun/psych_analysis/2.review/PICALM_CPM_df_2_plot_AD_control_genotype_NG2022.xlsx&quot;,sheet=3)
colnames(df) &lt;- c(&quot;sample&quot;,&quot;Diag&quot;,&quot;Genotype&quot;,&quot;eQTL_diagnosis&quot;,&quot;PICALM_expression&quot;,&quot;average_expr_group&quot;)

df &lt;- df[,1:5]

df &lt;- df %&gt;%
  mutate(G_dosage = case_when(
    Genotype == &quot;AA&quot; ~ 0,
    Genotype == &quot;AG&quot; ~ 1,
    Genotype == &quot;GG&quot; ~ 2,
    TRUE ~ NA_real_
  ))

DT::datatable(df,caption = htmltools::tags$caption( style = &#39;caption-side: left; text-align: left; color:black;  font-size:150% ;&#39;,&#39;Individual level PICALM expression&#39;),options = list(pageLength = 5) )</code></pre>
<div id="htmlwidget-212d6734bb3be6d86890" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-212d6734bb3be6d86890">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: left; text-align: left; color:black;  font-size:150% ;\">Individual level PICALM expression<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69"],["FM_010","FM_062","FM_024","FM_077","FM_030","FM_104","FM_078","FM_006","FM_048","FM_069","FM_046","FM_080","FM_054","FM_089","FM_092","FM_022","FM_102","FM_020","FM_064","FM_032","FM_086","FM_091","FM_002","FM_021","FM_095","FM_034","FM_007","FM_088","FM_083","FM_108","FM_059","FM_040","FM_005","FM_094","FM_028","FM_009","FM_050","FM_012","FM_033","FM_014","FM_081","FM_037","FM_087","FM_049","FM_038","FM_066","FM_103","FM_041","FM_073","FM_053","FM_058","FM_004","FM_105","FM_045","FM_008","FM_109","FM_098","FM_065","FM_015","FM_084","FM_076","FM_044","FM_099","FM_085","FM_093","FM_018","FM_051","FM_082","FM_001"],["Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","Alzheimer_Disease","dementia","dementia","dementia","dementia","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control"],["GG","GG","GG","AG","GG","GG","GG","GG","GG","GG","AG","GG","GG","GG","AG","GG","AA","GG","AA","GG","AA","AG","GG","AA","AA","AG","GG","AG","AG","AG","AG","AG","AG","AG","GG","AG","AG","AA","AG","AG","GG","GG","GG","GG","GG","AG","GG","GG","AG","GG","GG","AG","AG","AG","GG","AA","AA","GG","AA","AG","AG","AG","GG","AA","AG","GG","AG","GG","GG"],["AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","AD","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control","control"],[186.278052888828,208.086441242281,236.844722944341,256.346661073869,258.860603825908,276.139942674349,302.842338644568,313.388049291673,317.909733094421,323.000616146664,325.897809038708,328.542839807512,329.338818465779,335.429678105231,342.05420149336,359.366896372907,361.509955275409,361.717492572872,368.537586469047,371.988759693822,372.360843005822,374.563793870952,375.776381964953,376.918580638926,378.766154016086,379.767891952099,380.005070015464,380.463408019377,383.231062279911,384.003001831946,391.668265253079,394.80821642996,399.649172101082,411.748232277333,419.985293938525,436.216695654719,554.769567140323,577.574336692577,289.414235601814,331.252581571705,368.032262046405,419.553148763201,215.800516780814,259.648338239639,263.847335460793,269.231672431472,274.588348750972,276.79055382195,287.198955946775,292.57999874822,295.290103597087,296.183897201743,300.585582724645,308.143452081476,312.804620437753,317.26864782541,339.692657671785,350.67275987153,355.640241278102,366.260085613719,377.432686439113,396.846748549868,421.374472371175,442.69485853767,452.166363305057,458.813992909122,558.548299716376,611.656876024176,713.654810736527],[2,2,2,1,2,2,2,2,2,2,1,2,2,2,1,2,0,2,0,2,0,1,2,0,0,1,2,1,1,1,1,1,1,1,2,1,1,0,1,1,2,2,2,2,2,1,2,2,1,2,2,1,1,1,2,0,0,2,0,1,1,1,2,0,1,2,1,2,2]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>sample<\/th>\n      <th>Diag<\/th>\n      <th>Genotype<\/th>\n      <th>eQTL_diagnosis<\/th>\n      <th>PICALM_expression<\/th>\n      <th>G_dosage<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":[5,6]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="differential-expression-analysis-of-picalm"
class="section level1">
<h1>Differential expression analysis of PICALM</h1>
<ol style="list-style-type: decimal">
<li>The dataset was divided into 2 groups, according to the
<code>eQTL_diagnosis</code> column:</li>
</ol>
<ul>
<li>Alzheimer Disease (AD) samples</li>
<li>Non-Alzheimer Disease (nonAD) samples</li>
</ul>
<p>The black triangle represents the mean value of expressions in each
group</p>
<pre class="r"><code>mean_values &lt;- aggregate(PICALM_expression ~ eQTL_diagnosis, data = df, FUN = mean)

# Plot with mean values added
ggplot(df, aes(x = eQTL_diagnosis, y = PICALM_expression, color = eQTL_diagnosis)) +
  geom_boxplot() +
  geom_jitter(width = 0.2) +
  geom_point(data = mean_values, aes(x = eQTL_diagnosis, y = PICALM_expression), 
             color = &quot;black&quot;, size = 3, shape = 17) +  # Add mean points
  ggtitle(&quot;PICALM Expression by Group&quot;) +
  xlab(&quot;Genotype&quot;) +
  ylab(&quot;PICALM Expression&quot;) +
  theme_minimal()</code></pre>
<p><img src="figure/geno_expr_cor.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<ol start="2" style="list-style-type: decimal">
<li>Since the expression in AD samples is normally distributed</li>
</ol>
<pre class="r"><code>hist(df$PICALM_expression[df$eQTL_diagnosis ==&quot;AD&quot;], main = &quot;Distribution of PICALM_expression in AD samples&quot;)</code></pre>
<p><img src="figure/geno_expr_cor.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>the significance can be assessed by</p>
<div id="two-sample-t-test" class="section level2">
<h2>two sample t-test</h2>
<pre class="r"><code>t_test_result &lt;- t.test(PICALM_expression ~ eQTL_diagnosis, data = df)
print(t_test_result)</code></pre>
<pre><code>
    Welch Two Sample t-test

data:  PICALM_expression by eQTL_diagnosis
t = -0.30786, df = 39.566, p-value = 0.7598
alternative hypothesis: true difference in means between group AD and group control is not equal to 0
95 percent confidence interval:
 -58.34252  42.92267
sample estimates:
     mean in group AD mean in group control 
              355.824               363.534 </code></pre>
<pre class="r"><code>ggplot(df, aes(x = eQTL_diagnosis, y = PICALM_expression, color = eQTL_diagnosis)) +
  geom_boxplot() +
  geom_jitter(width = 0.2) +
  ggtitle(&quot;PICALM Expression by Group&quot;) +
  xlab(&quot;Genotype&quot;) +
  ylab(&quot;PICALM Expression&quot;) +
  theme_minimal()</code></pre>
<p><img src="figure/geno_expr_cor.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="annova" class="section level2">
<h2>ANNOVA</h2>
<pre class="r"><code>anova_result &lt;- aov(PICALM_expression ~ eQTL_diagnosis, data = df)
summary(anova_result)</code></pre>
<pre><code>               Df Sum Sq Mean Sq F value Pr(&gt;F)
eQTL_diagnosis  1    977     977   0.114  0.736
Residuals      67 572177    8540               </code></pre>
</div>
<div id="linear-regression" class="section level2">
<h2>Linear regression</h2>
<p>we tested PICALM_expression ~ eQTL_diagnosis</p>
<pre class="r"><code>model &lt;- lm(PICALM_expression ~ eQTL_diagnosis, data = df)
summary(model)</code></pre>
<pre><code>
Call:
lm(formula = PICALM_expression ~ eQTL_diagnosis, data = df)

Residuals:
    Min      1Q  Median      3Q     Max 
-169.55  -62.95    2.73   28.18  350.12 

Coefficients:
                      Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)             355.82      14.26  24.954   &lt;2e-16 ***
eQTL_diagnosiscontrol     7.71      22.80   0.338    0.736    
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 92.41 on 67 degrees of freedom
Multiple R-squared:  0.001704,  Adjusted R-squared:  -0.0132 
F-statistic: 0.1144 on 1 and 67 DF,  p-value: 0.7363</code></pre>
<ol start="3" style="list-style-type: decimal">
<li>Since mean value testing didn’t perform well, we used other testing
stats to test the median values</li>
</ol>
</div>
<div id="mann-whitney-u-test-wilcoxon-rank-sum-test"
class="section level2">
<h2>Mann-Whitney U Test (Wilcoxon Rank-Sum Test)</h2>
<p>The Mann-Whitney U test compares the median values of two independent
groups. It assesses whether values in one group tend to be larger or
smaller than values in another group.</p>
<pre class="r"><code>wilcox.test(PICALM_expression ~ eQTL_diagnosis, data = df)</code></pre>
<pre><code>
    Wilcoxon rank sum exact test

data:  PICALM_expression by eQTL_diagnosis
W = 631, p-value = 0.4375
alternative hypothesis: true location shift is not equal to 0</code></pre>
</div>
</div>
<div id="genotype-expression-correlation-of-picalm"
class="section level1">
<h1>Genotype-expression correlation of PICALM</h1>
<ol style="list-style-type: decimal">
<li>The dataset was divided into three groups, according to the
<code>eQTL_diagnosis</code> column:</li>
</ol>
<ul>
<li>All samples</li>
<li>Alzheimer Disease (AD) samples</li>
<li>Non-Alzheimer Disease (nonAD) samples</li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>Correlation Analysis:</li>
</ol>
<p>For each group (all samples, AD, and nonAD), the correlation between
PICALM expression and genotype was calculated using</p>
<ul>
<li>Linear regression: Expr ~ genotype, we used G dosage as
genotype</li>
</ul>
<pre class="r"><code>df_ad &lt;- df %&gt;% dplyr::filter(eQTL_diagnosis == &quot;AD&quot;)
df_nonad &lt;- df %&gt;% dplyr::filter(eQTL_diagnosis != &quot;AD&quot;)

print(&quot;The group for all samples&quot;)</code></pre>
<pre><code>[1] &quot;The group for all samples&quot;</code></pre>
<pre class="r"><code>model_all &lt;- lm(PICALM_expression ~ G_dosage, data = df)
summary(model_all)</code></pre>
<pre><code>
Call:
lm(formula = PICALM_expression ~ G_dosage, data = df)

Residuals:
    Min      1Q  Median      3Q     Max 
-155.13  -59.41  -14.93   29.29  372.24 

Coefficients:
            Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)   393.70      23.04  17.084   &lt;2e-16 ***
G_dosage      -26.14      15.23  -1.717   0.0907 .  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 90.52 on 67 degrees of freedom
Multiple R-squared:  0.04213,   Adjusted R-squared:  0.02783 
F-statistic: 2.947 on 1 and 67 DF,  p-value: 0.09066</code></pre>
<pre class="r"><code>sprintf(&quot;the p-value is %s&quot;, round(summary(model_all)$coefficients[2,4], digits = 4))</code></pre>
<pre><code>[1] &quot;the p-value is 0.0907&quot;</code></pre>
<pre class="r"><code>print(&quot;The group for AD samples&quot;)</code></pre>
<pre><code>[1] &quot;The group for AD samples&quot;</code></pre>
<pre class="r"><code>model_ad &lt;- lm(PICALM_expression ~ G_dosage, data = df_ad)
summary(model_ad)</code></pre>
<pre><code>
Call:
lm(formula = PICALM_expression ~ G_dosage, data = df_ad)

Residuals:
     Min       1Q   Median       3Q      Max 
-140.042  -41.684    3.503   34.809  184.194 

Coefficients:
            Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)   414.83      22.01   18.85  &lt; 2e-16 ***
G_dosage      -44.26      14.56   -3.04  0.00416 ** 
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 67.25 on 40 degrees of freedom
Multiple R-squared:  0.1876,    Adjusted R-squared:  0.1673 
F-statistic:  9.24 on 1 and 40 DF,  p-value: 0.004163</code></pre>
<pre class="r"><code>sprintf(&quot;the p-value is %s&quot;, round(summary(model_ad)$coefficients[2,4], digits = 4))</code></pre>
<pre><code>[1] &quot;the p-value is 0.0042&quot;</code></pre>
<pre class="r"><code>print(&quot;The group for non-AD samples&quot;)</code></pre>
<pre><code>[1] &quot;The group for non-AD samples&quot;</code></pre>
<pre class="r"><code>model_nonad &lt;- lm(PICALM_expression ~ G_dosage, data = df_nonad)
summary(model_nonad)</code></pre>
<pre><code>
Call:
lm(formula = PICALM_expression ~ G_dosage, data = df_nonad)

Residuals:
    Min      1Q  Median      3Q     Max 
-148.71  -73.89  -44.32   45.33  349.15 

Coefficients:
            Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)  361.589     47.884   7.551 6.62e-08 ***
G_dosage       1.459     31.600   0.046    0.964    
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 118.2 on 25 degrees of freedom
Multiple R-squared:  8.522e-05, Adjusted R-squared:  -0.03991 
F-statistic: 0.002131 on 1 and 25 DF,  p-value: 0.9635</code></pre>
<pre class="r"><code>sprintf(&quot;the p-value is %s&quot;, round(summary(model_nonad)$coefficients[2,4], digits = 4))</code></pre>
<pre><code>[1] &quot;the p-value is 0.9635&quot;</code></pre>
<pre class="r"><code># Convert G_dosage back to factor for plotting
df &lt;- df %&gt;% mutate(Genotype = factor(G_dosage, levels = c(0, 1, 2), labels = c(&quot;AA&quot;, &quot;AG&quot;, &quot;GG&quot;)))
df_ad &lt;- df_ad %&gt;% mutate(Genotype = factor(G_dosage, levels = c(0, 1, 2), labels = c(&quot;AA&quot;, &quot;AG&quot;, &quot;GG&quot;)))
df_nonad &lt;- df_nonad %&gt;% mutate(Genotype = factor(G_dosage, levels = c(0, 1, 2), labels = c(&quot;AA&quot;, &quot;AG&quot;, &quot;GG&quot;)))

# Plot for all samples
ggplot(df, aes(x = Genotype, y = PICALM_expression, color = Genotype)) +
  geom_boxplot() +
  geom_jitter(width = 0.2) +
  ggtitle(&quot;PICALM Expression by Genotype (All Samples)&quot;) +
  xlab(&quot;Genotype&quot;) +
  ylab(&quot;PICALM Expression&quot;)+
  theme_minimal()</code></pre>
<p><img src="figure/geno_expr_cor.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Plot for AD samples
ggplot(df_ad, aes(x = Genotype, y = PICALM_expression, color = Genotype)) +
  geom_boxplot() +
  geom_jitter(width = 0.2) +
  ggtitle(&quot;PICALM Expression by Genotype (AD Samples)&quot;) +
  xlab(&quot;Genotype&quot;) +
  ylab(&quot;PICALM Expression&quot;)+
  theme_minimal()</code></pre>
<p><img src="figure/geno_expr_cor.Rmd/unnamed-chunk-9-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Plot for non-AD samples
ggplot(df_nonad, aes(x = Genotype, y = PICALM_expression, color = Genotype)) +
  geom_boxplot() +
  geom_jitter(width = 0.2) +
  ggtitle(&quot;PICALM Expression by Genotype (Non-AD Samples)&quot;) +
  xlab(&quot;Genotype&quot;) +
  ylab(&quot;PICALM Expression&quot;) +
  theme_minimal()</code></pre>
<p><img src="figure/geno_expr_cor.Rmd/unnamed-chunk-9-3.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.2.0 (2022-04-22)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: CentOS Linux 7 (Core)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.3.13-el7-x86_64/lib/libopenblas_haswellp-r0.3.13.so

locale:
[1] C

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] dplyr_1.1.4   ggplot2_3.5.1

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.12       highr_0.9         cellranger_1.1.0  pillar_1.9.0     
 [5] compiler_4.2.0    bslib_0.3.1       later_1.3.0       jquerylib_0.1.4  
 [9] git2r_0.30.1      workflowr_1.7.0   tools_4.2.0       digest_0.6.29    
[13] jsonlite_1.8.0    evaluate_0.15     lifecycle_1.0.4   tibble_3.2.1     
[17] gtable_0.3.0      pkgconfig_2.0.3   rlang_1.1.2       cli_3.6.1        
[21] rstudioapi_0.13   crosstalk_1.2.0   yaml_2.3.5        xfun_0.41        
[25] fastmap_1.1.0     withr_2.5.0       stringr_1.5.1     knitr_1.39       
[29] htmlwidgets_1.5.4 generics_0.1.2    fs_1.5.2          vctrs_0.6.5      
[33] sass_0.4.1        DT_0.22           tidyselect_1.2.0  rprojroot_2.0.3  
[37] grid_4.2.0        glue_1.6.2        R6_2.5.1          fansi_1.0.3      
[41] readxl_1.4.0      rmarkdown_2.25    farver_2.1.0      magrittr_2.0.3   
[45] whisker_0.4       scales_1.3.0      promises_1.2.0.1  htmltools_0.5.2  
[49] colorspace_2.0-3  httpuv_1.6.5      labeling_0.4.2    utf8_1.2.2       
[53] stringi_1.7.6     munsell_0.5.0    </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
